app/routes/login.tsx

```diff
@@ -15,10 +15,45 @@ export const action = async ({ request }: ActionArgs) => {
 
 export default function Login() {
   return (
-    <Form method="post">
-      <Heading>Login</Heading>
-      <Text>You will be logged in (a session will be created for you)</Text>
-      <Button type="submit">Login</Button>
-    </Form>
+    <Center height="container.md" marginTop="28">
+      <Grid templateColumns="repeat(2, 1fr)">
+        <GridItem width="lg" height="xs">
+          <Box color="brand.500">
+            <Heading size="3xl">My</Heading>
```

Comment:
Instead of these waterfall boxes, can you do the same `my.custom.domain` concept here?

Reply:
Like a straight line "My.Custom.Domain"?

Reply:
Yes, straight line vs. waterfall.  I'm not a fan of the waterfall.

Reply:
![image](https://user-images.githubusercontent.com/32577022/218106231-a6d25b24-9078-43cb-a402-345ec1196e4a.png)

I've made changes with the layout, and made it vertical align instead to make it more straight to the point when a user views the page

Reply:
Nice!  For now, what if we use the `LockIcon` for the logo, so we have something?

<img width="188" alt="Screenshot 2023-02-10 at 9 47 41 AM" src="https://user-images.githubusercontent.com/427398/218120817-079a3cd2-fd8b-4012-a7d0-40a53bb0afcc.png">